tests/test_config.py

```diff
@@ -94,17 +90,16 @@ def test_load_ssl_config_cert_without_key_raises(cert_pem_file):
 
 
 def test_load_ssl_config_no_verify():
-    ssl_config = SSLConfig(verify=False)
-    context = ssl_config.ssl_context
+    context = httpx.create_ssl_context(verify=False)
     assert context.verify_mode == ssl.VerifyMode.CERT_NONE
     assert context.check_hostname is False
 
 
 def test_load_ssl_context():
     ssl_context = ssl.create_default_context()
-    ssl_config = SSLConfig(verify=ssl_context)
+    context = httpx.create_ssl_context(verify=ssl_context)
 
-    assert ssl_config.ssl_context is ssl_context
+    assert context is ssl_context
 
 
 def test_ssl_repr():
```

Comment:
We could probably drop the `__repr__` and `__eq__` methods from `SSLConfig`, and then drop these two tests.
Given that we're not exposing the instance anyplace they're pretty much superfluous.